src/main/java/heat/db/User.java

```diff
@@ -0,0 +1,65 @@
+package heat.db;
+
+import heat.dto.Gender;
+
+import java.util.Date;
+
+public class User {
```

Comment:
In `heat.db` classes like `ClassName` and in `heat.dto` like `ClassNameDto`. Mb it's more clear?

Reply:
Done.